package/yast2-storage.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Thu Feb  2 16:42:01 CET 2017 - shundhammer@suse.de
+
+- Move snapshots combo box to edit partition dialog (Fate#321094)
```

Comment:
combo box => check box

On the other hand, the text could be more clear. At first sight in "to edit partition dialog" may be interpreted as "move checkbox in order to edit x". Not sure if I did explain it correctly, but some people may need a second read.